tools/cli/pkg/command/upgrade_cluster.go

```diff
@@ -0,0 +1,60 @@
+package command
+
+import (
+	"fmt"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/common/orchestration"
+	"github.com/kyma-project/control-plane/tools/cli/pkg/logger"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type UpgradeClusterCommand struct {
+	UpgradeCommand
+	cobraCmd *cobra.Command
+}
+
+func NewUpgradeClusterCommand() *cobra.Command {
```

Comment:
Don't forget to construct the command in https://github.com/kyma-project/control-plane/blob/main/tools/cli/pkg/command/upgrade.go#L37

Reply:
https://github.com/kyma-project/control-plane/blob/aa81d0d6454b3221eb9d482327c9a3972beadf74/tools/cli/pkg/command/upgrade.go#L38